chart/README.md

```diff
@@ -36,7 +36,7 @@ cluster using the [Helm](https://helm.sh) package manager.
 
 ## Features
 
-* Supported executors: ``LocalExecutor``, ``CeleryExecutor``, ``KubernetesExecutor``, ``LocalKubernetesExecutor``, ``CeleryKubernetesExecutor``
+* Supported executors: ``LocalExecutor``, ``CeleryExecutor``, ``KubernetesExecutor``
```

Comment:
No, `CeleryKubernetesExecutor` and `LocalKubernetesExecutor` are supported but we just need to pass full path instead of just class names

Reply:
Note, again, only supported on 2.x. On Airflow >=3 they are not supported.